docs/Developer-Tips.md

```diff
@@ -47,3 +47,112 @@ Then to regenerate the Java code, run
 ```bash
 $ bin/tachyon protoGen
 ```
+
+### Full list of the commands in bin/tachyon
+
+Most commands in `bin/tachyon` are for developers. The following table explains the meaning and the
+syntax of each command.
+
+<table class="table table-striped">
+<tr><th>Command</th><th>Args</th><th>Meaning</th></tr>
+<tr>
+  <td>format</td>
+  <td>[-s]</td>
+  <td>Format Tachyon Master and all Workers. The option [-s] indicates that only format when underfs
+  is local and doesn't already exist.</td>
+</tr>
+<tr>
+  <td>formatWorker</td>
+  <td>None</td>
+  <td>Format Tachyon Worker storage on this local node.</td>
+</tr>
+<tr>
+  <td>bootstrap-conf</td>
+  <td>&lt;TACHYON_MASTER_HOSTNAME&gt;</td>
+  <td>Generate the bootstrap config file `tachyon-env.sh` with the specified
```

Comment:
Based on the image you showed, it looks like the backticks aren't monospacing `tachyon-env`. Is another approach needed when inside a `<td>` element?